### PR TITLE
1782 add provider enrichment update

### PR DESCRIPTION
**Pull request by defong**
### Context
Add provider enrichments via update endpoint

### Changes proposed in this pull request
Ability to add provider enrichments via update endpoint

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by defong** [comment]
[[M] Add provider update action to update enrichments](https://trello.com/c/EPoZ5ir9/1782-m-add-provider-update-action-to-update-enrichments)

**Comment by defong** [comment]
[[M] Add provider update action to update enrichments](https://trello.com/c/EPoZ5ir9/1782-m-add-provider-update-action-to-update-enrichments)

**Comment by defong** [comment]
[[M] Add provider update action to update enrichments](https://trello.com/c/EPoZ5ir9/1782-m-add-provider-update-action-to-update-enrichments)

**Comment by defong** [comment]
[[M] Add provider update action to update enrichments](https://trello.com/c/EPoZ5ir9/1782-m-add-provider-update-action-to-update-enrichments)

File: app/serializers/api/v2/serializable_provider_enrichment.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module API
+  module V2
+    class SerializableProviderEnrichment < JSONAPI::Serializable::Resource
+      type 'provider_enrichment'
+
+      has_one :provider
+
+      attributes  :email, :website, :address1, :address2, :address3, :address4,
```
Comment (dankmitchell) [inline app/serializers/api/v2/serializable_provider_enrichment.rb]: What about `accrediting_body` enrichments?

Comment (defong) [reply app/serializers/api/v2/serializable_provider_enrichment.rb]: argh #644 was meant to come after this pr :face_with_head_bandage:

Comment (timabell) [reply app/serializers/api/v2/serializable_provider_enrichment.rb]: https://trello.com/c/lhmNp5HF/1808-about-your-accredited-body-pull-onto-rails-backend

**Pull request merged**